string name
string action
string[] params
int8[] values
---
bool result